target/bin/delmailuser

```diff
@@ -144,17 +144,18 @@ use 'sudo docker exec mailserver rm -R /var/mail/${DOMAIN}/${USER}'"
   then
     if rm -R "/var/mail/${DOMAIN}/${USER}"
     then
-      echo "Mailbox deleted."
+      _log 'info' 'Mailbox deleted'
     else
-      echo "Mailbox couldn't be deleted." >&2
+      _log 'error' 'Mailbox could not be deleted' >&2
       ERROR=true
     fi
-    rmdir "/var/mail/${DOMAIN}" 2>/dev/null || true
+    rm -rf "/var/mail/${DOMAIN}"
   else
-    echo "Mailbox directory '/var/mail/${DOMAIN}/${USER}' did not exist." >&2
+    log 'error' "Mailbox directory '/var/mail/${DOMAIN}/${USER}' did not exist" >&2
```

Comment:
This occurs multiple times in this file, the lines have been changed to log with `warn`/`error` status, but you retained the `>&2` at the end of the original echo command? These two log levels already do `>&2` internally right?

Reply:
I overlooked this :) Thanks for noting it!

Reply:
Done in 08a05debd67e69c3e5010be547c0cd0ca65a28c7